spm_results_export.m

```diff
@@ -25,8 +25,13 @@ function spm_results_export(SPM,xSPM,TabDat)
 if ~isfield(SPM, 'filepath')
     outdir  = pwd;
 else
-    if exist(fullfile(SPM.filepath,'html')) ~= 7
-        mkdir(SPM.filepath,'html')
+    if exist(fullfile(SPM.filepath,'html')) ~= 7 
+        mkdir(SPM.filepath,'html');
+    else
+        if xSPM.nidm.oW == 1

```

Comment:
Can we add an `else` close here to deal with the case when the user asked not to overwrite the existing folder. In that case we could throw an error and quit (similarly to what is done in the NIDM exporter [here](https://github.com/incf-nidash/nidmresults/blob/master/nidmresults/exporter.py#L50-L52)). This could even happen earlier (as soon as we know there is already an `html` folder and that the user refuses to overwrite).